front/js/shared/yandex.es6

```diff
@@ -19,24 +18,48 @@
     fullEmail: 'info@shopelectro.ru',
   };
 
+  // @todo #129 Implement tracking of certain actions on front-end for YA and GA.
+  //  Actions: one-click purchase, changing products count on order page.
+  //  See the parent issue for a detail.
+
+  // Sync container for yaTracker
+  window.dataLayer = window.dataLayer || [];
+  // Load ecommerce plugin for gaTracker
+  ga('require', 'ecommerce');
+
+  let yaTracker = new YATracker(window.dataLayer, 'RUB');
+  let gaTracker = new GATracker(ga, 'ecommerce');
+
   const init = () => {
     setUpListeners();
+    // e-commerce container
+    // https://yandex.ru/support/metrika/data/e-commerce.html
   };
 
   function setUpListeners() {
     mediator.subscribe('onOneClickBuy', () => {
       reachGoal('CMN_BUY_SEND');
       reachGoal('FAST_BUY_SEND');
     });
-    mediator.subscribe('onOrderSend', () => {
+    mediator.subscribe('onOrderSend', (_, products) => {
       reachGoal('CMN_BUY_SEND');
       reachGoal('FULL_BUY_SEND');
+      // Use a dummy order's id, because we do not wait complete processing of
+      // purchase request.
+      yaTracker.purchase(products, {id: 'DummyId'});
+      gaTracker.purchase({id: 'DummyId'}, products);
+    });
+    mediator.subscribe('onProductAdd', (_, id, count) => {
+      yaTracker.add([{id: id, quantity: count}]);
```

Comment:
we should reach goal on product_add.
We have two different goals `PUT_IN_CART_FROM_PRODUCT` and `PUT_IN_CART_FROM_CATEGORY`.

If some diff code already do it, leave comment here plz

Reply:
@duker33 , yes, we publish onProductAdd event from both pages

Reply:
@artemiy312 , code needs comment too :)